### DISCUSSION TITLE
登陆页面在哪

### DISCUSSION THREAD

**Discussion by snowden-fu** [lobehub/lobehub#1922] [Self Hosting] [answered]
我已经部署好了，但是还没添加auth0服务，请问lobechat登陆页面是自带的吗？

**Comment by cy948** [comment, marked as answer]
配置 Auth0 服务：https://chat-docs.lobehub.com/zh/self-hosting/advanced/sso-providers/auth0 。另外，目前没有单独的登陆页面，只是当发起对话时会提示。
